src/index.js

```diff
@@ -551,17 +549,31 @@ function createConfig(options, entry, format, writeMeta) {
 							],
 						],
 					}),
-					// Only used for async await
 					babel({
-						// We mainly use bublé to transpile JS and only use babel to
-						// transpile down `async/await`. To prevent conflicts with user
-						// supplied configurations we set this option to false. Note
-						// that we never supported using custom babel configs anyway.
-						babelrc: false,
```

Comment:
so if someone has a babelrc in their project dir it now gets included?

Reply:
Yes, it's actually intentional. I'll add a few tests for it.

This way people can add babel-plugin-macros and other plugins. So it will work with every codebase. I like microbundle so I don't have to configure rollup and such but would love babel configuration. We could put it under an option and warn people.

Reply:
If you add a `.babelrc` that would overwrite the default config right? In that case it would be nice to have a way to just extend the default config so you don't need to add `preset-env` again etc. 

Next.js solves this in a neat way imo. And in our case it could look something like this

```js
// .babelrc
{
  "presets": "microbundle/babel",
  "plugins": [
    // my custom plugins
  ]
}
```

Reply:
yeah that would be ideal